README.md

```diff
@@ -107,7 +108,7 @@ Notable differences between ACCP and ACCP-FIPS:
 
 ACCP-FIPS is only supported on the following platforms:
 
-| Platfrom | FIPS support since version |
+| Platform | FIPS support since version |
```

Comment:
while we're in here, let's update our description of SecureRandom on L85 as AWS-LC [no longer defaults to jitter-based entropy](https://github.com/aws/aws-lc/blob/b2bbd9c287d3fff4e209892d342ea3b0d562477e/CMakeLists.txt#L25). I'd suggest something more generic, deferring documentation to AWS-LC:

```
* ACCP's SecureRandom uses [AWS-LC's DRBG implementation](https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/rand/rand.c).
```

Reply:
Sounds good, just changed in latest commit.